.github/workflows/homebrew.yml

```
@@ -32,41 +19,182 @@ jobs:
       - name: Install dependencies
         run: gem install octokit faraday
 
-      - name: Fetch latest release tag
+      - name: Find latest release with asd.phar
         id: latest_release
         run: |
```

actionlint (1.7.7)

workflow command "set-output" was deprecated. use `echo "{name}={value}" >> $GITHUB_OUTPUT` instead: https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions

(deprecated-commands)